components/scream/src/physics/p3/atmosphere_microphysics.cpp

```diff
@@ -96,12 +96,13 @@ void P3Microphysics::set_grids(const std::shared_ptr<const GridsManager> grids_m
   add_field<Updated> ("T_prev_micro_step",  scalar3d_layout_mid, K,        grid_name, ps);
 
   // Diagnostic Outputs: (all fields are just outputs w.r.t. P3)
-  add_field<Computed>("precip_liq_surf",    scalar2d_layout,     m/s,    grid_name);
-  add_field<Computed>("precip_ice_surf",    scalar2d_layout,     m/s,    grid_name);
+  add_field<Computed>("precip_liq_surf",    scalar2d_layout,     kg,     grid_name);
+  add_field<Computed>("precip_ice_surf",    scalar2d_layout,     kg,     grid_name);
   add_field<Computed>("eff_radius_qc",      scalar3d_layout_mid, micron, grid_name, ps);
   add_field<Computed>("eff_radius_qi",      scalar3d_layout_mid, micron, grid_name, ps);
 
   // History Only: (all fields are just outputs and are really only meant for I/O purposes)
+  // TODO: question - should these be averaged over subcycle as well?
```

Comment:
Yeah, these should also be averaged over subcycles. If we don't have bandwidth for doing that now, you could just delete them (since they are only used for output) and create a git issue to re-add them correctly. Though I guess that takes re-wiring... Just creating a git issue and putting a warning comment in the code here would be fine.

Reply:
I should be able to add them without too much difficulty.

Reply:
Ok.

Reply:
I just realized, there is no easy mechanism yet to zero these out every timestep.  So maybe we add a warning here in the comments and a git-issue to resolve it later.

Reply:
Ah yes, because these variables aren't used by surface coupling! How logical!!! Yeah, let's just add a warning in the code and a v1.1 issue in git.